datamade/just-spaces
# Remove Regina from keyring

## Overview

Closes #159.

Make sure that Regina is removed from the Blackbox keyring (😢)

## Testing Instructions

 * Checkout the branch
* Make sure you can still decrypt files
* Make sure Regina is off the keyring
